pkg/alertmanager/multitenant.go

```diff
@@ -22,23 +23,28 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promauto"
 
 	"github.com/cortexproject/cortex/pkg/alertmanager/alerts"
+	"github.com/cortexproject/cortex/pkg/ring"
+	"github.com/cortexproject/cortex/pkg/ring/kv"
 	"github.com/cortexproject/cortex/pkg/tenant"
 	"github.com/cortexproject/cortex/pkg/util"
 	"github.com/cortexproject/cortex/pkg/util/flagext"
 	"github.com/cortexproject/cortex/pkg/util/services"
 )
 
-var backoffConfig = util.BackoffConfig{
```

Comment:
I think this might be effectively a breaking change. Before, we would wait for all configurations to load indefinitely when starting up, whereas now we wouldn't and simply abort if we fail to do the initial sync. 

I find the new behaviour a bit better as it is clearer whenever the service would start or not but I'm open to reverting this back if we think this might be too complex to communicate back to our users.

Reply:
I don't think this is a breaking change. Changes like this happen frequently and, to me, are just enhnancements.